index.bs

```diff
@@ -1649,10 +1652,11 @@ Each {{XRWebGLLayer}} MUST have a <dfn>list of viewports</dfn> which is a [=/lis
 
 <div class="algorithm" data-algorithm="get-viewport">
 
-The <dfn method for="XRWebGLLayer">getViewport(|view|)</dfn> method, when invoked, MUST run the following steps:
+The <dfn method for="XRWebGLLayer">getViewport(|view|)</dfn> method, when invoked on an {{XRSession}} |session|, MUST run the following steps:
```

Comment:
`getViewport()` is a method of an `XRWebGLLayer`, so saying "invoked on an XRSession" is incorrect. The `XRWebGLLayer` accepts an `XRSession` in the constructor, and that's the one that we should be checking against here, but it turns out that we're not explicitly keeping a reference to that? We do check it in [the `updateRenderState()` algorithm](https://immersive-web.github.io/webxr/#dom-xrsession-updaterenderstate), but it kinda cheats:

> If newState’s baseLayer's was created with an XRSession other than session, throw an InvalidStateError and abort these steps.

We could do the same thing here and just reference "the XRSession the layer was created with" or we could actually start keeping an explicit reference to it. I'll leave that to you.

Reply:
Oh, for some reason i've been thinking that it's a method on XRSession all this time!

Reply:
Done!